packages/cli/yaml/yaml-schema/src/schemas/ExampleResponseSchema.ts

```diff
@@ -1,9 +1,73 @@
 import { z } from "zod";
 import { ExampleTypeReferenceSchema } from "./ExampleTypeReferenceSchema";
 
-export const ExampleResponseSchema = z.strictObject({
+/**
+ * response: unknown
+ * examples:
+ *  - name: example 1
+ *    request:
+ *      stream: false
+ *    response:
+ *      body: {}
+ *
+ */
+export const ExampleBatchResponseSchema = z.strictObject({
     error: z.optional(z.string()),
     body: z.optional(ExampleTypeReferenceSchema)
 });
 
+export type ExampleBatchResponseSchema = z.infer<typeof ExampleBatchResponseSchema>;
+
+/**
+ * response-stream:
+ *   type: unknown
+ * examples:
+ *  - name: example 1
+ *    request:
+ *      stream: true
+ *    response:
+ *      stream:
+ *        - {} # event 1
+ *        - {} # event 2
+ */
+export const ExampleStreamResponseSchema = z.strictObject({
+    stream: z.array(ExampleTypeReferenceSchema)
+});
+
+export type ExampleStreamResponseSchema = z.infer<typeof ExampleStreamResponseSchema>;
+
+export const ExamplSseEventSchema = z.strictObject({
+    event: z.string(),
+    data: z.optional(ExampleTypeReferenceSchema)
+});
+
+export type ExamplSseEventSchema = z.infer<typeof ExamplSseEventSchema>;
+
+/**
+ * response-stream:
+ *   type: unknown
+ *   format: sse
+ * examples:
+ * - name: example 1
+ *   request:
+ *     stream: true
+ *   response:
+ *     stream:
+ *      - event: chat # event 1
+ *        data: {}
+ *      - event: chat # event 2
+ *        data: {}
+ */
+export const ExampleSseResponseSchema = z.strictObject({
+    stream: z.array(ExamplSseEventSchema)
+});
+
+export type ExampleSseResponseSchema = z.infer<typeof ExampleSseResponseSchema>;
+
+export const ExampleResponseSchema = z.union([
+    ExampleBatchResponseSchema,
+    ExampleStreamResponseSchema,
+    ExampleSseResponseSchema
+]);
```

Comment:
can we add an `sse` test defnition [here](https://github.com/fern-api/fern/tree/main/test-definitions/fern/apis) to make sure this works.